.circleci/config.yml

```diff
@@ -211,38 +198,59 @@ jobs:
 
                   # At this stage, we can only do canary tests when CONFIG_ENV=test as the electron binary is not signed
                   # TODO: We might be able to ignore this once we switched to electron-builders auto-update
-                  if [ -n "${CALYPSO_HASH}" ]; then
+                  if [ "$CONFIG_ENV" == "test" ]; then
                     source $HOME/.nvm/nvm.sh
                     nvm use
                     make test
                   fi
       - *calypso_finalize_cache
       - *calypso_save_cache
-      - persist_to_workspace:
-          root: /Users/distiller/wp-desktop
-          paths: *app_cache_paths
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.14.0-browsers
+    working_directory: ~/wp-desktop
+    environment:
+      CONFIG_ENV: test
+      MINIFY_JS: false
+      NODE_ARGS: --max_old_space_size=2048
+    steps:
+      - checkout
+      - *install_linux_dependencies
+      - build-and-test
       - *webhook_notify_success
       - *webhook_notify_failed
 
+  build-release:
+    macos:
+      xcode: "9.0"
+    working_directory: ~/wp-desktop
+    environment:
+      CONFIG_ENV: release
+      MINIFY_JS: true
+      NODE_ARGS: --max_old_space_size=8192
+    steps:
+      - checkout
+      - build-and-test
+      - persist_to_workspace:
+          root: ~/wp-desktop
+          paths: *app_cache_paths
+
   linux:
     docker:
-      - image: electronuserland/builder:wine-mono
-    working_directory: /wp-desktop
+      - image: circleci/node:10.14.0-browsers
```

Comment:
Harmless, but this build uses `nvm` to get the node version Calypso wants. We could likely use a lighter image.

Reply:
Yes, we possibly could but the reason I am using this image is because of the `-browsers` functionaltiy which enable us to headless browser testing. Without this, we would have to do lots of boilerplate configuration ourselves.